app/process-image.js

```diff
@@ -643,15 +656,17 @@ class ImageProcessing {
 							reject(err);
 						}
 
-						this.tesseract.recognize(image)
+						this.time_tesseract.recognize(image, this.time_remain_tesseract_options)
 							.catch(err => reject(err))
 							.then(result => {
-								// NOTE: important that the letter "o" be replaced with a 0, in order to properly match a time
-								const match = result.text.replace(/[^\w\s:]/g, '').replace(/[oO]/g, 0).match(/([0-9]{1,2}:[0-9]{1,2}){2}/g);
+								const confident_words = this.tesseractGetConfidentSequences(result, true),
+									match = confident_words.length > 0 ?
+										confident_words[0].match(/(\d{1,2}:\d{2}:\d{2})/) :
+										'';
 								if (match && match.length) {
 									resolve({
 										image: new_image,
-										text: match[0],
+										text: match[1],
```

Comment:
Same thing here, a little odd that it's the 2nd matched result instead of the 1st, just would like to understand this better